src/libraries/System.IO.FileSystem/tests/FileInfo/SymbolicLinks.cs

```diff
@@ -41,6 +41,24 @@ protected override void AssertIsCorrectTypeAndDirectoryAttribute(FileSystemInfo
         public void ResolveLinkTarget_Throws_NotExists() =>
             ResolveLinkTarget_Throws_NotExists_Internal<FileNotFoundException>();
 
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void UnsupportedLink_ReturnsNull()
+        {
+            string unsupportedLinkPath = GetAppExecLinkPath();
+            if (unsupportedLinkPath is null)
```

Comment:
In a case like this, I'd recommend you throw SkipTestException instead of return true. That is more truthful, and also makes it easy to confirm whether the test is running by simply looking at the skip count or the test log. Otherwise you have to crack open the debugger.

Not asking you to change this today but did you verify that the tests are getting past here?

Reply:
Easiest is probably to change it locally to throw that way. (And you could submit that later to main)

Reply:
Yes, I tested locally that the tests are actually passing this check and reaching the assertions below. 
If this works for me locally I assume that there must be at least one CI leg capable of completing the whole scenario past this check.

Reply:
Very likely, but not always true (eg., tests that require the machine or user is a member of a domain)

Good for now then.